### PR TITLE
migrate(obc): css co classes to odf specific

### PR DESCRIPTION
**Pull request by debjyoti-pandit**
**Current**
image

______________________________________________________________________________________

**Current**
image

_______________________________________________________________________________________

**Current**
image

_______________________________________________________________________________________

**Current**
image

### CODE REVIEW

File: packages/odf/style.scss [hunk 1]
```diff
@@ -1 +1,3 @@
+@import '../shared/styles/vars';
+@import '../shared/styles/common';
```
Comment (GowthamShanmugam) [inline packages/odf/style.scss:2]: Please check style.scss in shared, which will resolve this vars import

Comment (GowthamShanmugam) [reply packages/odf/style.scss:2]: just importing style.scss from shared is enough

Comment (debjyoti-pandit) [reply packages/odf/style.scss]: Done

**Review by GowthamShanmugam — APPROVED** [review]

**Comment by debjyoti-pandit** [comment]
This latest push doesn't contain any new changes, I have only rebased the branch with master.

**Review by bipuladh — APPROVED** [review]
/lgtm

**Pull request merged**